client/scss/components/_blockquote.scss

```diff
@@ -1,3 +1,5 @@
+//* @define blockquote; weak
```

Comment:
a `// @define some-component-name` block at the top of each component's scss partial ensures that all class names follow the `block__element--modifier` naming convention.